.gitignore

```diff
@@ -2,6 +2,7 @@
 guide/book
 target
 vendor
+.idea
```

Comment:
Please keep this ignore. 🥲 

jetbrains IDEs always tries to create a dir `.idea` in a working dir.
This would make you environment clean.